src/pip/_internal/commands/check.py

```diff
@@ -20,6 +22,7 @@ class CheckCommand(Command):
       %prog [options]"""
 
     def run(self, options, args):
+        # type: (Values, List[Any]) -> Any
```

Comment:
```suggestion
        # type: (Values, List[Any]) -> int
```

With `return SUCCESS` at the end of the method.

Reply:
I think this have to do with a bigger issue: https://github.com/pypa/pip/blob/e060970d51c5946beac8447eb95585d83019582d/src/pip/_internal/cli/base_command.py#L199-L203

Although I do agree that we should gradually clean up the hints of correct cases.

Reply:
Thanks @sbidoul and @McSinyx for your comments. 

My understanding was that we have to add mypy annotations to the code without touching the implementation. If that's not the case, and perhaps @pradyunsg can add to it too, I will update the code with your suggestions in the next commit.

Reply:
> add mypy annotations to the code without touching the implementation

Generally, yes, that is less risky. On the other hand I think that goal must be balanced with using meaningful typing annotation, which means avoiding `Any` as much as possible.

So I'd say making a trivial change that goes towards the goal of having all `run()` methods return an int makes sense.

Reply:
True @sbidoul , I have made the necessary changes.